pkg/tinydtls/doc.txt

```diff
@@ -4,4 +4,33 @@
  * @ingroup  net
  * @brief    Provides the Eclipse TinyDTLS to RIOT
  * @see      https://projects.eclipse.org/projects/iot.tinydtls
- */
\ No newline at end of file
+ *
+ * Usage
+ * -----
+ *
+ * Add as a package in your application Makefile:
```

Comment:
`your application's Makefile` or `Makefile of your application`

Reply:
Fixed.